### PR TITLE
[Truffle] improve j+tr Readme

### PR DESCRIPTION
**Pull request by pitr-ch**

### CODE REVIEW

File: tool/truffle/jruby_truffle_runner/README.md [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
-`rubygems` and `bundler`
+`jruby+truffle_runner` gem is a small command line utility designed to run and
+test Ruby gems and applications on JRuby+Truffle Ruby runtime. It uses other
+Ruby implementation to prepare environment and to execute files, tests
+on JRuby+Truffle. It is a temporary tool to make working with JRuby+Truffle
+easy until it fully supports `rubygems` and `bundler`
 
 ## Installation
 
+It assumes that you have other Ruby available, if you have JRuby's git
```
Comment (chrisseaton) [inline tool/truffle/jruby_truffle_runner/README.md]: 'have *another* Ruby available'

File: tool/truffle/jruby_truffle_runner/README.md [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+repository clone it can be compiled and used to run this tool. Rbenv is quite
+good tool to install and manage different Rubies. To add yours compiled JRuby
```
Comment (chrisseaton) [inline tool/truffle/jruby_truffle_runner/README.md]: 'to add *your*'

File: tool/truffle/jruby_truffle_runner/README.md [hunk 3]
```diff
@@ -1,20 +1,29 @@
...
 
 1.  Install the gem from rubygems.org (Not yet published).
     Install the gem `gem install jruby+truffle_runner`
 2.  Use `jt` tool from JRuby's repository.
     Run `jt install-tool` to install the gem from the cloned repository.
-3.  Create alias for the gem's executable. E.g. add 
-    `alias jruby+truffle="~/path-to-jruby/tool/truffle/jruby_truffle_runner/bin/jruby+truffle" to your `.zshrc`.
+3.  Create alias for the gem's executable. E.g. add
+    `alias jruby+truffle="~/path-to-jruby/tool/truffle/jruby_truffle_runner/bin/jruby+truffle"`
+    to your `.zshrc`.
```
Comment (chrisseaton) [inline tool/truffle/jruby_truffle_runner/README.md]: Can't assume people are running zshrc - is even the default anywhere?

File: tool/truffle/jruby_truffle_runner/README.md [hunk 4]
```diff
@@ -25,29 +34,92 @@ There is a `setup` subcommand to create environment for JRuby+Truffle.
...
 
-It copies default configuration (part of the gem) if one is available for a given gem (looks for a `gemspec` in current 
-directory). It installs all required gems (based on `Gemfile`) to local bundle (default path: `.jruby+truffle_bundle`), 
-and executes other steps defined in configuration file (`.jruby+truffle.yaml`) or as command line options 
-(see `jruby+truffle setup --help` to learn what additional setup steps are available, or see one of the bundled 
-configuration files found in `gem_configurations` directory).
+It copies default configuration (part of the gem) if one is available for a
+given gem (it looks for a `gemspec` in current directory). It installs all
+required gems (based on `Gemfile`) to local bundle (default path:
+`.jruby+truffle_bundle`), and executes other steps defined in configuration
```
Comment (chrisseaton) [inline tool/truffle/jruby_truffle_runner/README.md]: 'in *the* configuration file'

File: tool/truffle/jruby_truffle_runner/README.md [hunk 5]
```diff
@@ -25,29 +34,92 @@ There is a `setup` subcommand to create environment for JRuby+Truffle.
...
+file (`.jruby+truffle.yaml`) or as command line options (see `jruby+truffle
+setup --help` to learn what additional setup steps are available, or see one of
+the bundled configuration files found in `gem_configurations` directory). After
+it finishes `run` subcommand can be used.
```
Comment (chrisseaton) [inline tool/truffle/jruby_truffle_runner/README.md]: 'After it finishes, *the* `run` subcommand can be used'

**Comment by chrisseaton** [comment]
Looks good. A few English language typos.

**Comment by pitr-ch** [comment]
Thanks, merging.

**Pull request merged**